src/scripts/settings/userSettings.js

```diff
@@ -239,6 +255,33 @@ export class UserSettings {
         return val === 'true';
     }
 
+    /**
+     * Get or set 'disableCustomCss' state.
+     * @param {boolean|undefined} val - Flag to enable 'disableCustomCss' or undefined.
+     * @return {boolean} 'disableCustomCss' state.
+     */
+    disableCustomCss(val) {
+        if (val !== undefined) {
+            return this.set('disableCustomCss', val.toString(), false);
+        }
+
+        val = this.get('disableCustomCss', false);
+        return val === 'true';
```

Comment:
I think this will calm SonarCloud
```suggestion
        return this.get('disableCustomCss', false) === 'true';
```